### PR TITLE
Fix navbar using Vuetify

### PR DESCRIPTION
**Pull request by keelywan**
## Navbar 
This makes the navbar more compatible with the other Vuetify components. It should fix the problem that occurs when you need to scroll the page and the other components (e.g. search bar) appear on top of the navbar.

This is a screenshot of what the page looks like when you scroll down a little - the search bar doesn't appear above the navbar, but is covered by it.

navbar-scroll

### CODE REVIEW

**Review by andres-holguin — APPROVED** [review]
Oh, awesome! I was just about to start this when the PR came in :D Got a screenshot?

**Comment by keelywan** [comment]
> Oh, awesome! I was just about to start this when the PR came in :D Got a screenshot?

Yep, I just added it to the description above. It's just a screenshot of what it looks like when the page has been scrolled down a little. Is that what you were looking for?

**Comment by andres-holguin** [comment]
> > Oh, awesome! I was just about to start this when the PR came in :D Got a screenshot?
> 
> Yep, I just added it to the description above. It's just a screenshot of what it looks like when the page has been scrolled down a little. Is that what you were looking for?

Perfect, nice work.

**Review by Aradhya-B — APPROVED** [review]
Good stuff!!

File: src/main/webapp/components/Navbar.vue [hunk 1]
```diff
@@ -1,58 +1,67 @@
 <template>
-  <nav class="navy-blue sticky nav-bar">
-    <a href="index.html" class="bold" id="title">starfish</a>
-    <img src="/assets/starfish.png" id="logo">
-    <div class="action-items" v-if="!signedIn">
+  <div style="height:80px">
```
Comment (nickKelman) [inline src/main/webapp/components/Navbar.vue]: Prefer using class styling and scss over inline styling. Better to have it grouped in one spot

**Review by nickKelman — APPROVED** [review]

File: src/main/webapp/components/Navbar.vue (hunk 1 above)
Comment (keelywan) [reply src/main/webapp/components/Navbar.vue]: Ok I'll change that

**Pull request merged**